### ISSUE TITLE
Unhelpful Error Message When Invalid Device URI Specified

### ISSUE THREAD

**Issue by michaelrsweet** [apple/cups#4003]
Version: 1.5-current
CUPS.org User: gsexton

So I was trying to use CUPS on OpenSUSE 12.1 after upgrading from OpenSUSE 11.4

Cups did not work because the lp module was not loaded.

The error message in the /var/log/cups/error_log file was

Unable to open print file: no such file or directory.

This lead me to believe the problem was a permission problem on the spooler directory.

It would have been VASTLY more helpful if the error message had been:

Unable to open print file parallel:/dev/lp0: no such file or directory.

Eventually, I got the linux lp module loading. However, in my edits I put the device URI in as:

parallel://dev/lp0

again things didn't work. 

So, in conclusion, many hours of CUPS troubleshooting could be resolved if the error message were just a tiny bit more specific and gave people some sort of idea as to what the problem really was.
